index.d.ts

```diff
@@ -36,6 +36,46 @@ export interface Hooks {
 	afterResponse?: AfterResponseHook[];
 }
 
+export interface RetryOptions {
+	/**
+ 	Number of retries to retry failed requests
+
+ 	@default 2
+
+ 	@example
+ 	```
+ 	import ky from 'ky';
+
+ 	const parsed = await ky('https://example.com', {retry: {retries: 10, methods: ['get'], statusCodes: [413]}}).json();
+ 	```
+ 	*/
+	retries?: number;
+	/**
+ 	The set of methods allowed to retry
+
+ 	@default new Set(['get', 'put', 'head', 'delete', 'options', 'trace'])
+ 	*/
+	methods?: Set<string> | Array<string>;
```

Comment:
Only array as input.

```suggestion
	methods?: string[];
```